samples/mstest-runner/NativeAotRunner/nuget.config

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <!-- MSTest early access packages. See: https://aka.ms/mstest/preview uncomment this line -->
-    <!-- <add key="test-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json" /> -->
+     <clear />
+     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
We should not do that, this is causing security alerts. We should instead use the `dotnet-public` feed. I think just dropping this `nuget.config` to use the one from parent folder would be enough.